docs/data/_index.md

```diff
@@ -160,10 +160,16 @@ Sync data stored on the machine to the cloud.
 For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/services/datamanager).
 
 ```go {class="line-numbers linkable-line-numbers"}
-data, err := datamanager.FromRobot(robot, "my_data_service")
```

Comment:
You should be able to do this once https://github.com/viamrobotics/rdk/pull/3891 is merged. That will be the only way to call `Sync` on a data manager service from what I can tell.